.github/workflows/test-many-projects.yml

```diff
@@ -0,0 +1,119 @@
+
+name: auto-run-test-many-projects-script
+
+on: [push]
+
+jobs:
+  manyProjTest1:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Check if it's possible to have some way to describe the same steps across all jobs, since all of them use the same setup.